src/main/java/Duke.java

```diff
@@ -1,10 +1,102 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static void greetUser() {
+         String divider = "______________________________";
+         String greeting = divider + System.lineSeparator() + "Hello! I'm Jarvis!"
+                 + System.lineSeparator() + "What can I do for you?"
+                 + System.lineSeparator() + divider + System.lineSeparator();
+         System.out.println(greeting);
+    }
+
+    public static void echoUserInput(String curr) {
+        String divider = "______________________________";
```

Comment:
This is a constant used throughout the code. Using a named constant will make it easier to maintain code since you just need to edit once for it to apply to rest of the code.
Something like "public static final String LINE_DIVIDER "______________________________"; " instead might be good.
https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers